bukuserver/static/bukuserver/css/bookmark.css

```diff
@@ -9,3 +9,18 @@ table tr td:not(:first-child) {
         width: min(1000px, 80%);
     }
 }
+
+.link {
+    display: block;
+}
+
+.tag-list {
+    font-size: larger;
+    display: flex;
+    gap: 1px;
+    flex-wrap: wrap;
+}
+
+.description {
+    white-space: pre-wrap;
+}
```

Comment:
All these are blocks now; `.tag-list` also now has gaps between its elements (`font-size:` is for badges), and `.description` retains whitespace as-is (other than wrapping long lines)
![bookmarks list](https://github.com/user-attachments/assets/74f96f88-88aa-4be1-b55f-673cf41cd447)
(note: the last "netloc" is actually a tag)